tests/ctst/steps/pra.ts

```diff
@@ -161,7 +162,7 @@ async function waitForPhase(
     return false;
 }
 
-Given('a DR installed', { timeout: 360000 }, async function (this: Zenko) {
+Given('a DR installed', { timeout: 330000 }, async function (this: Zenko) {
```

Comment:
Should we specify the timeout for the install command, to match this one? By default it's 30min (so 1800000).